api/src/controllers/organization_invites.rs

```
@@ -93,7 +93,8 @@ pub fn create(
     }
     //If an active invite exists for this email then first expire it before issuing the new invite.
     if let Ok(i) = OrganizationInvite::find_active_invite_by_email(&email, &*connection) {
```

Are you not swallowing an error here?

---

Yes I am because it is equally valid that an email exists or doesn't exist depending on if this is the first invite or not. I only want to expire the previous invite IF there is an active invite and if there is none then I want to do nothing but it is not cause for throwing an error.